apps/src/studio/Item.js

```diff
@@ -28,8 +28,6 @@ var Item = function (options) {
   }
   Studio.trackedBehavior.createdItems[this.className]++;
 
-  this.spritesCounterclockwise = options.spritesCounterclockwise || false;
```

Comment:
not needed because Collidable constructor already applies all properties. And `undefined` is fine instead of `false` when it is not supplied.